packages/cyberstorm/src/components/Layout/Developers/ManifestValidator/ManifestValidatorLayout.tsx

```diff
@@ -1,20 +1,142 @@
 "use client";
 import styles from "./ManifestValidatorLayout.module.css";
 import { SettingItem } from "../../../SettingItem/SettingItem";
-import { TextInput } from "../../../TextInput/TextInput";
 import { BreadCrumbs } from "../../../BreadCrumbs/BreadCrumbs";
 import { BaseLayout } from "../../BaseLayout/BaseLayout";
 import { ManifestValidatorLink } from "../../../Links/Links";
 import { PageHeader } from "../../BaseLayout/PageHeader/PageHeader";
+import { useState, useEffect } from "react";
+import { CodeInput } from "../../../CodeInput/CodeInput";
+import { Alert } from "../../../Alert/Alert";
+import { Icon } from "../../../Icon/Icon";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faWarning } from "@fortawesome/pro-solid-svg-icons";
+import { Select } from "../../../Select/Select";
+
+interface HTMLResponse {
+  success?: string;
+  non_field_errors?: string[];
+  namespace?: string[];
+}
+
+export const isRecord = (obj: unknown): obj is Record<string, unknown> =>
```

Comment:
If this same type guard (or others) start to pop up in multiple places, consider creating `utils/type_guards.ts` oslt and collecting them there.

Reply:
Added separate type_guards.ts